axlearn/common/inference_test.py

```diff
@@ -168,13 +169,10 @@ def predict_batch(self, input_batch: NestedTensor) -> NestedTensor:
 def is_supported(
     platform: str,
     mesh_shape: tuple[int, int],
-    param_dtype: jnp.dtype,
     inference_dtype: Optional[jnp.dtype],
     global_batch_size: int,
     data_partition: DataPartitionType,
-    use_ema: bool = False,
 ):
-    del param_dtype, use_ema  # not used
```

Comment:
why are we changing this?

Reply:
we don't need these args to check for skipping